aztec/src/main/kotlin/org/wordpress/aztec/AztecParser.kt

```diff
@@ -200,6 +214,23 @@ object AztecParser {
                     if (spans[j] is CommentSpan) {
                         out.append("<!--")
                     }
+
+                    if (spans[j] is HiddenHtmlSpan) {
+                        val span = spans[j]  as HiddenHtmlSpan
```

Comment:
With Kotlin we can rewrite this method to avoid casting.
Instead of accessing span through `spans[j]` every time, we can declare it at the beginning of the loop, something like `val span = spans[j]` and then, after checking `span` type with `is` condition, it will automatically be casted if true.